ci/playbooks/content_provider/pre.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: all
+  tasks:
+    - name: Clone repos in the job workspace
+      include_role:
+        name: prepare-workspace
+
+    - name: Install ansible-core
+      become: true
+      package:
+        name: ansible-core
+        state: present
```

Comment:
missing newline char - pre-commit and/or ansible-test will cry

Reply:
Done